src/core/controllers/map-state.js

```diff
@@ -75,12 +77,11 @@ export default class MapState extends ViewState {
       zoom,
       bearing,
       pitch,
-
-      altitude: ensureFinite(altitude, defaultState.altitude),
-      maxZoom: ensureFinite(maxZoom, MAPBOX_LIMITS.maxZoom),
-      minZoom: ensureFinite(minZoom, MAPBOX_LIMITS.minZoom),
-      maxPitch: ensureFinite(maxPitch, MAPBOX_LIMITS.maxPitch),
-      minPitch: ensureFinite(minPitch, MAPBOX_LIMITS.minPitch)
+      altitude,
+      maxZoom,
+      minZoom,
+      maxPitch,
+      minPitch
```

Comment:
why do we remove the range check?

Reply:
The old fallback mechanism is replaced by default parameters.